torch/csrc/cuda/Module.cpp

```diff
@@ -398,6 +399,13 @@ bool THCPModule_initCuda(PyObject *torch_module) {
   END_HANDLE_TH_ERRORS_RET(false)
 }
 
+cudaDeviceProp *getCudaDeviceProp(int device)
```

Comment:
Could also be a lambda, directly defined in `m.def("get_device_properties", ...)`